appengine/wordpress/src/files/flexible/app.yaml

```
@@ -2,7 +2,7 @@ runtime: php
 vm: true
 
 beta_settings:
-  cloud_sql_instances: {{project_id}}:us-central1:{{db_instance}}
+  cloud_sql_instances: {{project_id}}{{db_region}}{{db_instance}}
```

Yeah, I agree, but it's kind of complicated.

I want to fail fast (before downloading WordPress) so I want to check $sql_gen before that. On the other hand, when I check the environment (flexible/standard) we only download the app engine standard, but at that time, wordpress should be there, etc.

Anyways, point taken, I changed it to create a pattern for sprintf.